Doc/library/socket.rst

```diff
@@ -1511,7 +1511,7 @@ to sockets.
 
    .. availability:: most Unix platforms, possibly others.
 
-   .. versionadded:: 3.3
+   .. versionadded:: 3.8
```

Comment:
This versionadded note is for `sendmsg` - `send_fds` should get a separate documentation block now that it is its own function.